src/main/java/frc/robot/RobotContainer.java

```diff
@@ -137,7 +137,7 @@ public RobotContainer() {
     m_drive.setupModulesShuffleboard();
     m_vision.setupVisionShuffleboard();
     m_driver.setupShuffleboard();
-    m_vision.logging();
+    m_vision.logPose(m_drive.getPose());
```

Comment:
This dose nothing but add a pose of (0,0,0) to our logs, and the robot pose logging should be in drivetrain

Reply:
Read comment below.